src/apps/games/serializers/rating_game.py

```diff
@@ -49,6 +50,8 @@ class Meta:
         }
 
     def validate(self,data):
+        if data["white_network"] == data["black_network"]:
```

Comment:
Is it mandatory/good to do this logic twice ?

Reply:
Yes. There are some code paths within django itself (like admin page) that run clean() but do not run validate(). And django rest framework runs validate() but not clean().
And annoying is that the type is different in each case. In clean() you have a django model object instance. And in django rest framework, you have a dictionary of raw user post data.

Reply:
Ok, weird. I guess it is fine for now, maybe later we can optimize that.